data/benefits/javascript/martinique-pret-etudiant-taux-zero.yml

```
@@ -16,6 +14,10 @@ conditions:
     suivant l'obtention du diplôme préparé ou la fin des études.
   - Avoir remboursé intégralement la dette au plus tard six ans après la date de
     début du paiement de la créance.
+  - Télécharger <a
+    target="_blank" title="Formulaire de cautionnement solidaire - Nouvelle fenêtre" rel="noopener"
+    href="https://www.spot.mq/sites/default/files/Documents/SPOT_FORMULAIRE%20CAUTIONNEMENT.pdf">le formulaire de cautionnement solidaire
```

```suggestion
    href="https://www.spot.mq/sites/default/files/Documents/SPOT_FORMULAIRE%20CAUTIONNEMENT.pdf">le formulaire de cautionnement solidaire (PDF - 697ko).
```
C'est important de préciser qu'on va récupérer un PDF et d'indiquer le poids (démarche d'éco-conception).